docs/topics/signals.txt

```diff
@@ -12,30 +12,15 @@ has taken place. They're especially useful when many pieces of code may be
 interested in the same events.
 
 Django provides a :doc:`set of built-in signals </ref/signals>` that let user
-code get notified by Django itself of certain actions. These include some useful
-notifications:
+code get notified by Django itself of certain actions.
 
-* :data:`django.db.models.signals.pre_save` &
-  :data:`django.db.models.signals.post_save`
+This is an example of the ``setting_changed`` signal, which is used by many pluggable
+Django apps::
 
-  Sent before or after a model's :meth:`~django.db.models.Model.save` method
-  is called.
-
-* :data:`django.db.models.signals.pre_delete` &
-  :data:`django.db.models.signals.post_delete`
-
-  Sent before or after a model's :meth:`~django.db.models.Model.delete`
-  method or queryset's :meth:`~django.db.models.query.QuerySet.delete`
-  method is called.
-
-* :data:`django.db.models.signals.m2m_changed`
-
-  Sent when a :class:`~django.db.models.ManyToManyField` on a model is changed.
-
-* :data:`django.core.signals.request_started` &
-  :data:`django.core.signals.request_finished`
-
-  Sent when Django starts or finishes an HTTP request.
+    @receiver(setting_changed)
+    def reload_my_app_settings(**kwargs):
+        if kwargs['setting'] == 'MY_APP':
+            my_app_settings.reload()
```

Comment:
I think this should join the `ref` document, so those looking up `setting_changed` see the example.

Reply:
Done :)

Reply:
I think it's best to remove the copy here, to avoid duplication.

Reply:
Removed